enclave-runner/src/usercalls/mod.rs

```diff
@@ -714,9 +717,13 @@ impl EnclaveState {
                                     println!("Attaching debugger");
                                     #[cfg(all(unix, not(target_abi = "musl")))]
                                     trap_attached_debugger(usercall.tcs_address() as _).await;
-                                    Err(EnclaveAbort::Exit {
-                                        panic: EnclavePanic::from(buf.into_inner()),
-                                    })
+                                    let panic = EnclavePanic::from(buf.into_inner());
+                                    println!("{}", &panic);
```

Comment:
This print shouldn't be here

Reply:
I think I'm missing something. I thought the point was to print the panic message as soon as the runner received one from the enclave.

Reply:
Unless you mean that it should be inside the if

Reply:
I'm ok with it inside the if. But then it should be an `eprintln` and it should print something like `Enclave panicked: {}`

Reply:
That is perfectly reasonable. What do you think about panicking with that message instead of aborting the process?

Reply:
That seems fine but the option should be called `forward_panics`

Reply:
The prefix `Enclave panicked:` is already in the message produced by `EnclavePanic::from(buf.into_inner());` I think.

Reply:
Good point Mohsen, I will omit that then